### PR TITLE
Massachusetts short-term capital gains rate decrease

### LINKED ISSUE #3434: Massachusetts short-term capital gains rate decrease
Currently, Massachusetts imposes a 12% tax on short-term capital gains, representing profits on assets that are realized within a year of acquisition.
With the new [bill](https://malegislature.gov/Bills/193/H4104), the tax rate is now reduced to 8.5%, beginning in 2023.

### PR DESCRIPTION
**Pull request by Kiki-67**
Fixes #3434

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]:
```suggestion
- bump: patch
```

File: changelog_entry.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  changes:
+    added:
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:3]:
```suggestion
    fixed:
```

File: changelog_entry.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+    - Massachusetts short term capital gains.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - 2023 Massachusetts short term capital gains.
```

File: policyengine_us/parameters/gov/states/ma/tax/income/capital_gains/short_term_capital_gains_rate.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ma/tax/income/capital_gains/short_term_capital_gains_rate.yaml:1]: dont need new files, just add the 2023 value to the existing parameter (+ legal code and bill) and a new 2023 test

File: policyengine_us/variables/gov/states/ma/tax/income/ma_short_term_capital_gains.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ma/tax/income/ma_short_term_capital_gains.py:1]: delete

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml [hunk 4]
```diff
@@ -12,5 +13,7 @@ metadata:
       href: https://www.mass.gov/service-details/massachusetts-tax-rates
     - title: "Schedule B Interest, Dividends and Certain Capital Gains and Losses"
       href: https://www.mass.gov/doc/2021-schedule-b-interest-dividends-and-certain-capital-gains-and-losses/download
+    - title: Bill H.4104 Section 8
+      href: https://malegislature.gov/Bills/193/H4104/BillHistory#page=4
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml:17]:
```suggestion
      href: https://malegislature.gov/Bills/193/H4104/BillHistory
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml [hunk 5]
```diff
@@ -1,6 +1,7 @@
 description: The tax rate on Part A short-term capital gains.
 values:
   2002-01-01: 0.12
+  2023-01-01: 0.085
 metadata:
   label: MA short-term capital gains rate
   reference:
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml:7]: can you change the below one to `(a)(1)` and leave a comment indicating that it does not yet reflect the 8.5% change as of 2023-12-22?

File: policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml [hunk 6]
```diff
@@ -1,6 +1,7 @@
 description: The tax rate on Part A short-term capital gains.
 values:
   2002-01-01: 0.12
+  2023-01-01: 0.085
 metadata:
   label: MA short-term capital gains rate
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml:6]:
```suggestion
  label: Massachusetts short-term capital gains tax rate
```

File: policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml [hunk 7]
```diff
@@ -1,6 +1,7 @@
 description: The tax rate on Part A short-term capital gains.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ma/tax/income/rates/part_a/capital_gains.yaml:1]:
```suggestion
description: Massachusetts taxes Part A short-term capital gains at this rate.
```

File: policyengine_us/tests/policy/baseline/gov/states/ma/tax/income/ma_income_tax_before_credits.yaml [hunk 8]
```diff
@@ -24,3 +24,12 @@
     state_code: MA
   output:
     ma_income_tax_before_credits: 140_000
+
+- name: 8.5% rate on capital gains.
```
Comment (MaxGhenis) [inline policyengine_us/tests/policy/baseline/gov/states/ma/tax/income/ma_income_tax_before_credits.yaml:28]:
```suggestion
- name: Massachusetts lowered its capital gains rate to 8.5% in 2023.
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**